scene/gui/graph_edit.cpp

```diff
@@ -920,7 +920,7 @@ void GraphEdit::_draw_connection_line(CanvasItem *p_where, const Vector2 &p_from
 		scaled_points.push_back(points[i] * p_zoom);
 	}
 
-	p_where->draw_polyline_colors(scaled_points, colors, Math::floor(p_width * get_theme_default_base_scale()), lines_antialiased);
+	p_where->draw_polyline_colors(scaled_points, colors, MAX(p_width, Math::floor(p_width * get_theme_default_base_scale())), lines_antialiased);
```

Comment:
```suggestion
	// Thickness below 0.5 doesn't look good on the graph or its minimap.
	p_where->draw_polyline_colors(scaled_points, colors, MAX(0.5, Math::floor(p_width * get_theme_default_base_scale())), lines_antialiased);
```

I think we should have a fixed number here, because it is possible, in theory, that the width supplied to this method would be bigger than `1` and the theme scale would be smaller than `1.0`). Since we can handle it as low as 0.5 without visual artifacts, I think we can limit it by this value specifically.

And the comment there helps to explain why the value was picked to a future reader :)

Reply:
Done!